mscore/timedialog.cpp

```diff
@@ -127,7 +127,6 @@ void TimeDialog::save()
 
 void TimeDialog::zChanged(int val)
       {
-      zText->setText(QString("%1").arg(val));
```

Comment:
I actually don't really care one way or the other as I'm unlikely to do this, but maybe it makes to still set the text *if* it is already set?  Or perhaps only if it also matches the old value?  Otherwise, if for whatever reason (like, an imported score that was set this way) you have text already there, then you change the value using the UI controls, the text isn't updated.  On the other hand, maybe that's OK if you had put some special text there you don't want cleared.

Reply:
This just makes it unnecessarily complicated. The texts should only be there for special occasions where written values should somehow be _different_ as internal values, it's their only purpose. I think any attempt of making the texts the _same_ as internal values is an overreaction. If a text is already there, even if it's the same as the internal value, it has to be treated as a special text that is not connected with the latter, because not doing so completely misses the point.

Reply:
And please note that this fix only affects the "Create Time Signatures" dialogue, not the "Time Signature Properties" one. So the imported-score problem is not real.

Reply:
Good point, I had them confused.  I was concerned about the case of existing scores where the text *is* set, and is the same as the actual value.  But of course, Time Signature Properties doesn't even have these controls.  So I agree, ignore my previous comment :-)